docs/src/components/home/keyFeaturesSection.tsx

```diff
@@ -39,23 +39,23 @@ function KeyFeaturesCardGrid(): JSX.Element {
 				bodyLabel="Easy to use"
 				bodyText={easyToUseText}
 				learnMoreLinkHref="/docs/start/quick-start"
-				learnMoreLinkAltText="Fluid Framework Quick Start"
+				learnMoreLinkAltText="Easy to use, Learn More"
 			/>
 			<KeyFeatureCard
 				imageSrc="https://storage.fluidframework.com/static/images/website/home/open-source.png"
 				imageAltText="Open source"
 				bodyLabel="Open Source"
 				bodyText={openSourceText}
 				learnMoreLinkHref="/community"
-				learnMoreLinkAltText="Community"
+				learnMoreLinkAltText="Open Source, Learn More"
 			/>
 			<KeyFeatureCard
 				imageSrc="https://storage.fluidframework.com/static/images/website/home/performance.png"
 				imageAltText="Industry-leading speed and performance"
 				bodyLabel="Industry-leading speed & performance"
 				bodyText={performanceText}
 				learnMoreLinkHref="/docs/build/dds#performance-characteristics"
-				learnMoreLinkAltText="Performance"
+				learnMoreLinkAltText="Industry-leading speed & performance, Learn More"
```

Comment:
Nit, although I don't know if it matters much for accessibility purposes.
```suggestion
				learnMoreLinkAltText="Industry-leading speed and performance, Learn More"
```

Reply:
Yeah, the voiceover/narrator still reads it as "and".